etna/authors/models.py

```diff
@@ -152,6 +152,23 @@ class AuthorTag(models.Model):
     )
 
 
+class AuthorSerializer(serializers.ModelSerializer):
+    title = serializers.CharField()
+    image = ImageRenditionField("fill-512x512|format-jpeg|jpegquality-60")
```

Comment:
I _think_ it might be:

```python
image_webp = ImageRenditionField("fill-600x400|format-webp|webpquality-80", source="image")
```

Reply:
Do we want `image` and `image_webp`? Or `image_jpeg` and `image_webp`?

Reply:
Let's go for `image_jpeg` and `image_webp`. I'm going to make them both (eventually all) return as `teaser_image_jpeg` and `teaser_image_webp` to keep things consistent when we're talking about the "image" for a page link. Otherwise we'll end up with all different sorts of images! 😵‍💫